### ISSUE TITLE
hcxdumptool Multiple interfaces?

### ISSUE THREAD

**Issue by PowerPress** [ZerBea/hcxdumptool#238]
Does hcxdumptool support using multiple interfaces of supported adapters working together?

**Comment by ZerBea** [comment]
No. I tried it and noticed that multiple interfaces interfere with each other.
Please read "Lessons learned (to be continued)" here:
https://github.com/ZerBea/wifi_laboratory

**Comment by ZerBea** [comment]
BTW:
There is no need to support more than one interface.
Just run hcxdumptool twice or more (either in different terminals or append & to the command):
$ sudo hcxdumpoll -i wlan0 -c 1
$ sudo hcxdumpoll -i wlan1 -c 6
$ sudo hcxdumpoll -i wlan2 -c 11
